spacy/language.py

```diff
@@ -951,6 +952,7 @@ def remove_pipe(self, name: str) -> Tuple[str, PipeCallable]:
         # Make sure the name is also removed from the set of disabled components
         if name in self.disabled:
             self._disabled.remove(name)
+        self._link_components()
```

Comment:
We should add a test for the `remove_pipe` functionality and double check that it's removed from the internal `listener_map` (which it probably wasn't before this PR)